.circleci/config.yml

```diff
@@ -355,7 +363,7 @@ references:
         name: Test
         command: |
           epmd -daemon
-          MAX_RETRIES=2 ./ci/try make ${MAKE_TARGET:?} CT_TEST_FLAGS="--suite=$(.circleci/split_suites.sh)"
+          ./ci/try make ${MAKE_TARGET:?} CT_TEST_FLAGS="--suite=$(.circleci/split_suites.sh)"
```

Comment:
Note that this may run different set of sets if it's run multiple times, I'm not aware how the CircleCI internal mechanics works and if they always return the same results for a commit hash or not.

That script runs:
```
circleci tests split --split-by=timings --timings-type=classname
```

The split is statistical based on latest run times.